src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -491,6 +516,10 @@ const ATSUAOCPage = () => {
     const { simbriefUserId, setSimbriefUserId } = useContext(SimbriefUserIdContext);
     const [simbriefDisplay, setSimbriefDisplay] = useState(simbriefUserId);
 
+    const [hoppieUserId, setHoppieUserId] = usePersistentProperty('CONFIG_HOPPIE_USERID');
+    const [hoppieError, setHoppieError] = useState(false);
+    const [hoppieEnabled, setHoppieEnabled] = useState(SimVar.GetSimVarValue('L:A32NX_HOPPIE_ACTIVE', 'number') === 1 ? 'ENABLED' : 'DISABLED');
```

Comment:
This is not a setting. It should be a Connect/Disconnect button, not in the settings pages. Probably fits best on the ATC page.

Reply:
Moved the connect-stuff to the ATC page.

![grafik](https://user-images.githubusercontent.com/64070348/152557951-7b63f1f3-d64e-45c1-98cb-c19569a009a4.png)